jib-core/src/main/java/com/google/cloud/tools/jib/registry/credentials/DockerCredentialHelperFactory.java

```diff
@@ -16,18 +16,29 @@
 
 package com.google.cloud.tools.jib.registry.credentials;
 
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
 /** Factory class for constructing {@link DockerCredentialHelper}. */
 public class DockerCredentialHelperFactory {
 
+  public static final String CREDENTIAL_HELPER_PREFIX = "docker-credential-";
+
   public DockerCredentialHelperFactory() {}
```

Comment:
I'm not sure I see the value in having a separate factory — when there's nothing to configure on the factory — vs static methods on `DockerCredentialHelper`.